### PR TITLE
Standardize 'requires' text in all help commands

### PR DESCRIPTION
**Pull request by panpawn**
This introduces Chat#require which accepts the lowest rank a command requires and then returns all ranks above it (excluding player) so that all help command 'requires' text will be standardized throughout PS.

### CODE REVIEW

**Comment by panpawn** [comment]
This is a WIP and not ready for merge.

**Comment by panpawn** [comment]
Okay, now it's ready.

**Comment by Zarel** [comment]
I don't really think this is an improvement?

**Comment by panpawn** [comment]
If a rank changes symbol for whatever reason, this will make it more streamlined and less hard-coded.

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1918,7 +1920,7 @@ exports.commands = {
 	},
 	htmlboxhelp: [
 		"/htmlbox [message] - Displays a message, parsing HTML code contained.",
-		"!htmlbox [message] - Shows everyone a message, parsing HTML code contained. Requires: ~ & #",
+		"!htmlbox [message] - Shows everyone a message, parsing HTML code contained. " + Chat.require('#'),
```
Comment (Zarel) [inline chat-plugins/info.js]: Like, this is more text than before!

One thing we could do is just `Requires: #` in the text, and then do a search and add higher ranks after it.

Comment (panpawn) [inline chat-plugins/info.js]: We could also shorten it to ``Chat#req``

Comment (Zarel) [inline chat-plugins/info.js]: ...no

**Comment by Morfent** [comment]
I think it'd be a better idea to do this when the commands are first imported, rather than on each command individually.. That way they can get parsed to a standard format all  at once without the need for another `Chat` method

**Comment by Zarel** [comment]
Closing. This is also weird because it's unclear when to include room ranks (e.g. RO, Player) in the list.

**Pull request closed without merging**